### PR TITLE
Improved nullability annotations, deleted unused code.

### PR DESCRIPTION
**Pull request by Elland**

### CODE REVIEW

File: Token/Controllers/Settings/Backup Phrase/BackupPhraseEnableController.swift [hunk 1]
```diff
@@ -94,10 +94,6 @@ class BackupPhraseEnableController: UIViewController {
         ])
     }
 
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-    }
-
```
Comment (roberthein) [inline Token/Controllers/Settings/Backup Phrase/BackupPhraseEnableController.swift]: ✨

File: Token/Models/ContactsManager.m [hunk 2]
```diff
@@ -16,11 +16,11 @@ - (Yap *)yap {
     return [Yap sharedInstance];
 }
 
-+ (BOOL)name:(NSString * _Nonnull)nameString matchesQuery:(NSString * _Nonnull)queryString {
++ (BOOL)name:(nonnull NSString *)nameString matchesQuery:(nonnull NSString *)queryString {
```
Comment (yuliaveres) [inline Token/Models/ContactsManager.m]: i think those are required only in headers, aren't they ? :)
but not a mistake anyway

Comment (Elland) [inline Token/Models/ContactsManager.m]: Is that so? I thought they would give me a warning b/c of mismatched nullability indicators?

File: Token/Supporting Files/AppDelegate.m [hunk 3]
```diff
@@ -326,7 +326,7 @@ - (void)updateRemoteNotificationCredentials {
             }
         }];
 
-    } failure:^(NSError * _Nonnull error) {
+    } failure:^(NSError * error) {
```
Comment (yuliaveres) [inline Token/Supporting Files/AppDelegate.m]: i am not sure about the style guide here, maybe *error ?

**Comment by yuliaveres** [comment]
looks good to me too 👍

File: Token/Supporting Files/AppDelegate.m (hunk 3 above)
Comment (Elland) [inline Token/Supporting Files/AppDelegate.m]: Strangely enough, we don't have one for ObjC 👍 But yeah, I usually do `Class *name`. Adjusting it.

File: Token/Models/ContactsManager.m (hunk 2 above)
Comment (yuliaveres) [inline Token/Models/ContactsManager.m]: no, it shouldn't ( as far as I remember)

**Pull request merged**